src/components/TreemapCell.tsx

```diff
@@ -76,6 +75,42 @@ extends DoubleClickReactComponent<TreemapCellProps, {}> {
     };
   }
 
+  constructor(props) {
```

Comment:
I really like the change from using TreemapRectangle and TreemapCell to just having this content in TreemapCell. Makes a lot more sense to me.